GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -87,6 +87,11 @@ public void Execute()
                 // any platform specific preparation required to run as a background process.
                 GVFSPlatform.Instance.PrepareProcessToRunInBackground();
             }
+
+            if (!GVFSPlatform.Instance.FileSystem.IsFileSystemSupported(this.EnlistmentRootPathParameter, out string errorMessage))

```

Comment:
A better spot for this check would be in `MountVerb` rather than `InProcessMountVerb`.  If we check in `MountVerb` the user will be able to see the error message in the console (`InProcessMountVerb` is running as a background process).

Reply:
I did think about that, but then if the service starts the process we won't see the check right?
It's such a rare case that the clone would success and the mount would not, I'm OK with the error being a bit harder to find. 

What do you think?

Reply:
> I did think about that, but then if the service starts the process we won't see the check right?

The service calls `gvfs mount` and so the check will still be run:

https://github.com/microsoft/VFSForGit/blob/d0336e0690e58fd86c478f2c53e896dc3bed3b6e/GVFS/GVFS.Service/GVFSMountProcess.Mac.cs#L24

> I'm OK with the error being a bit harder to find.  
> What do you think?

Since `gvfs mount` is always called when mounting I think it makes sense to make the check there.  

Making the errors easier to surface to users was one of the motivations for updating GVFS.Service to call `gvfs mount` rather than `GVFS.Mount` directly.

Reply:
@wilbaker I didn't realize we had moved the service to call gvfs mount'.  Thanks for the update there.  In that case we definitely want the check in MountVerb, updated.